src/main/java/dev/openfeature/sdk/MutableContext.java

```diff
@@ -33,7 +33,7 @@ public MutableContext(String targetingKey) {
     }
 
     public MutableContext(Map<String, Value> attributes) {
-        this("", attributes);
+        this("", new HashMap<>(attributes));
```

Comment:
In the `ImmutableContext` you set the `targetingKey` to null, here to `""`.

Reply:
made it null, I don't like the empty strings.